engine/getstarted/step_three.md

```diff
@@ -26,7 +26,7 @@ image you'll use in the rest of this getting started.
 
     ![Browse Docker Hub](tutimg/image_found.png)
 
-3. Click on the **docker/whalesay** image in the results.
+3. Click on the **docker/whalesay** image in the results. You'll find it about halfway down.
```

Comment:
The search results might change over time, so today it might be halfway down, but tomorrow can be somewhere else. Do you care to update your proposal with a direct link to the image instead? https://hub.docker.com/r/docker/whalesay/